languages/Chinese_Traditional.xml

```diff
@@ -1,195 +1,196 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
-<LanguageStrings version="1.5.5" Author="Tenjin">
+<?xml version="1.0" encoding="UTF-8"?>
+<LanguageStrings version="1.5.9" Author="Yi-Jyun Pan, Tenjin">
     <MainWindow>
-        <staticSensorsListHeader>偵測器列表：</staticSensorsListHeader>
+        <staticSensorsListHeader>溫度感測器清單：</staticSensorsListHeader>
         <listColumnFan>風扇</listColumnFan>
-        <listColumnRPM>最低/當前/最大旋轉速度</listColumnRPM>
-        <listColumnControl>控制設定</listColumnControl>
-        <listColumnSensor>偵測器</listColumnSensor>
-        <listColumnSensorValue>溫度</listColumnSensorValue>
-        <listSeparatorDrives>磁碟：</listSeparatorDrives>
-        <popupMenuItemControl>修改控制設定...</popupMenuItemControl>
-        <popupMenuItemReset>重設為自動控制</popupMenuItemReset>
-        <fanControlAutomatic>自動</fanControlAutomatic>
+        <listColumnRPM>最小/目前/最大轉速</listColumnRPM>
+        <listColumnControl>控制模式</listColumnControl>
+        <listColumnSensor>感測器</listColumnSensor>
+        <listColumnSensorValue>溫度讀數</listColumnSensorValue>
+        <listSeparatorDrives>磁碟機：</listSeparatorDrives>
+        <popupMenuItemControl>更改控制設定⋯</popupMenuItemControl>
+        <popupMenuItemReset>重設為自動控制模式</popupMenuItemReset>
+        <fanControlAutomatic>自動控制</fanControlAutomatic>
         <fanControlAutomaticInfo>自動（系統控制）</fanControlAutomaticInfo>
-        <fanControlConstant>固定轉速為 %1</fanControlConstant>
+        <fanControlConstant>固定 %1 轉速</fanControlConstant>
         <fanControlSensor>基於 %1</fanControlSensor>
         <fanControlUnknown>未知！</fanControlUnknown>
-        <buttonFanControlCustom>自定...</buttonFanControlCustom>
-        <buttonFanControlCustomInfo>指定自定控制模式</buttonFanControlCustomInfo>
+        <buttonFanControlCustom>客製化⋯</buttonFanControlCustom>
+        <buttonFanControlCustomInfo>客製化控制模式</buttonFanControlCustomInfo>
         <buttonMinimize>最小化</buttonMinimize>
-        <buttonMinimizeMac>隱藏到選單列</buttonMinimizeMac>
-        <buttonPreferences>設定...</buttonPreferences>
-        <buttonAbout>關於...</buttonAbout>
-        <messageHelperToolInstall>Macs Fan Control 需要安裝一個幫助程序來控制風扇. 如果你只是只想監視風扇溫度傳感器則可以跳過這個安裝.</messageHelperToolInstall>
-        <messageHelperToolUpgrade>Macs Fan Control需要安裝一個更新的版本的幫助程式來控制風扇.</messageHelperToolUpgrade>
-        <messageFanControlNotAvailableTitle>風扇控制不可用</messageFanControlNotAvailableTitle>
-        <messageFanControlNotAvailable>Macs Fan Control 無法連接至輔助程序以便開啟風扇控制並且調整其運轉速度。\n\n請確認每當要求的時候，務必要輸入管理員的密碼。\n\nb你仍然可以在監視模式下使用此應用程式.</messageFanControlNotAvailable>
```

Comment:
try to add the character "b" from "\nb" back? I'm not sure whether it is an additional or required character.